annoyed-alligators/socl_media/apps/chat/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+# Create your models here.
+
+
+class Chat(models.Model):
```

Comment:
I think you should just name this model `message`, since this stores the messages and will create a DB table for messages.